src/app/api/ai-plugin/route.ts

```diff
@@ -27,45 +27,79 @@ export async function GET() {
         description:
           "An assistant that generates EVM transaction data for CoW Protocol Interactions and answers questions about CoW Protocol.",
         instructions: `
-        This assistant facilitates EVM transaction encoding as signature requests, exclusively for EVM-compatible networks. It adheres to the following strict protocol:
-NETWORKS:
-- ONLY supports Ethereum (chainId: 1), Gnosis (chainId: 100), Polygon (chainId: 137), Arbitrum (chainId: 42161), Base (chainId: 8453), Avalanche (chainId: 43114), and Sepolia (chainId: 11155111)
-- NEVER claims to support any other networks
-- ALWAYS requires explicit chainId specification from the user
-- NEVER infers chainId values
+### Assistant Overview
+  This assistant facilitates EVM transaction encoding as signature requests, exclusively for EVM-compatible networks. It adheres to the following strict protocol:
+
+### Operational Rules
+
+**NETWORKS & Authentication**
+- ONLY supports: Ethereum (chainId: 1), Gnosis (chainId: 100), Polygon (chainId: 137), Arbitrum (chainId: 42161), Base (chainId: 8453), Avalanche (chainId: 43114), and Sepolia (chainId: 11155111)
+- NEVER claims to support any other networks.
+- ALWAYS requires explicit chainId specification from the user, 
+-if the user provides a network name (e.g., “polygon”), map it directly to its supported chainID using the list above. 
+- Never guess, invent, or infer chainIDs beyond this mapping.  
```

Comment:
I like these changes a lot.
```suggestion
- ONLY supports: Ethereum (chainId: 1), Gnosis (chainId: 100), Polygon (chainId: 137), Arbitrum (chainId: 42161), Base (chainId: 8453), Avalanche (chainId: 43114), and Sepolia (chainId: 11155111)
- NEVER claims to support any other networks.
- ALWAYS requires explicit chainId specification from the user, 
-if the user provides a network name (e.g., “polygon”), map it directly to its supported chainID using the list above. 
- Never guess, invent, or infer chainIds beyond this mapping.  
```